tests/unit/modules/test_fusion_az.py

```diff
@@ -0,0 +1,1193 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2023, Denys Denysyev (ddenysyev@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING.GPLv3 or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import fusion as purefusion
+
+from unittest.mock import Mock, MagicMock, patch
+import pytest
+
+from ansible_collections.purestorage.fusion.tests.unit.mocks.operation_mock import (
+    OperationMock,
+    OperationStatus,
+)
+from ansible_collections.purestorage.fusion.tests.unit.mocks.module_mock import (
+    ModuleMock,
+    ModuleSucceeded,
+)
+from ansible_collections.purestorage.fusion.tests.unit.mocks.exceptions_mock import (
+    ApiExceptionsMock,
+)
+from ansible_collections.purestorage.fusion.plugins.module_utils.errors import (
+    OperationException,
+)
+
+from urllib3.exceptions import HTTPError
+from ansible_collections.purestorage.fusion.plugins.modules import fusion_az
+
+fusion_az.install_fusion_exception_hook = MagicMock()
+fusion_az.get_fusion = MagicMock()
+current_module = (
+    "ansible_collections.purestorage.fusion.tests.unit.modules.test_fusion_az"
+)
+
+
+def default_module_az_params(state="present", display_name="foo_az"):
```

Comment:
This can be used as a fixture, so you don't need to call this function every time again.